prettier.config.js

```diff
@@ -1 +1,23 @@
-module.exports = require('@sumup/foundry/prettier').base;
+/**
+ * Copyright 2020, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module.exports = require('@sumup/foundry/prettier')(
+  {},
+  {
+    trailingComma: 'none',
+    arrowParens: 'avoid',
+    quoteProps: 'as-needed'
+  }
```

Comment:
These rules changed in Foundry v2. They'd cause a lot of stylistic changes in the source files, which would lead to many merge conflicts while we work on v1 and v2 in parallel. We can apply them once work on v1 has been stopped completely.